### ISSUE TITLE
Need to Test the Wink Plugin

### ISSUE THREAD

**Issue by schpeen** [hoobs-org/HOOBS#316]
**Description**
after installing plugin and following Wink developer app creation the same redirect URI i useed for Hoobs 2 it worked but on hoobs 3 I cannot get the redirect to work even pointing it at IP address. 

**Expected behavoir**
it should work like it did on Hoobs 2

**Plugin name**
Wink Hoobs Certified

**Version**
Hoobs 3.1.1

**Did you upgrade**
Please let us know if you upgraded from a previous version.
- [ ] Yes
- [ x] No

**Pervious version**
Tried that but migration failed so I started from scratch and installed Hoobs 3

**Did you orginally upgrade to HOOBS 3 from HOOBS 2.1.1?**
Did you perform an upgrade from HOOBS 2.1.1? Some things are different.
- [ ] Yes
- [ ]x No

**What device are you using?**
Please let us know the device you are running HOOBS on.
- [ ] HOOBS Box
- [ ] HOOBS SD Card
- [ x] Custom Install

**Post your config**
This can be found in the interface, Configuration -> Advanced or you can get it via SSH `cat ~/.hoobs/etc/config.json`

* Remove any passwords from the config file. This is public.

```
{
    "server": {
        "port": 8080,
        "autostart": 0,
        "home_setup_id": "xxxxxxxxxxxxx",
        "polling_seconds": 5
    },
    "client": {
        "default_route": "status",
        "inactive_logoff": 30,
        "theme": "hoobs-dark",
        "locale": "en",
        "temp_units": "fahrenheit",
        "country_code": "US",
        "postal_code": "xxxxxxx"
    },
    "bridge": {
        "name": "HOOBS",
        "port": 51826,
        "pin": "xxxxxxxxxxx",
        "username": "xxxxxxxxxxx"
    },
    "description": "",
    "ports": {},
    "accessories": [],
    "platforms": [
        {
            "platform": "Ring",
            "plugin_map": {
                "plugin_name": "ring"
            },
            "email": "xxxxxxxxx",
            "password": "xxxxxx",
            "hideUnsupportedServices": "true",
            "showPanicButtons": "true"
        },
        {
            "platform": "SmartThings",
            "plugin_map": {
                "plugin_name": "homebridge-smartthings-tonesto7"
            },
            "name": "SmartThings",
            "app_url": "https://graph-na02-useast1.api.smartthings.com:443/api/smartapps/installations/",
            "app_id": "xxxxx",
            "access_token": "xxxxx"
        },
        {
            "platform": "TplinkSmarthome",
            "plugin_map": {
                "plugin_name": "homebridge-tplink-smarthome"
            }
        },
        {
            "platform": "NeatoVacuumRobot",
            "plugin_map": {
                "plugin_name": "homebridge-neato"
            },
            "email": "xxxxx",
            "password": "xxx"
        },
        {
            "platform": "HarmonyHubWebSocket",
            "plugin_map": {
                "plugin_name": "homebridge-harmony"
            },
            "name": "Harmony Living Room",
            "hubIP": "192.168.1xxxxxxxx",
            "publishAllTVAsExternalAccessory": "true",
            "cleanCache": "false",
            "TVAccessory": "true",
            "publishGeneralMuteSwitch": "true",
            "publishDevicesAsIndividualAccessories": "true",
            "publishSwitchActivitiesAsIndividualAccessories": "true",
            "publishSequencesAsIndividualAccessories": "true",
            "switchAccessories": "true"
        },
        {
            "platform": "Wink",
            "plugin_map": {
                "plugin_name": "wink"
            },
            "name": "Wink",
            "client_secret": "xxxxxxxxxxxxxxxxxxxx",
            "client_id": "xxxxxxxxxxxxxxx",
            "username": "",
            "password": ""
        }``

**Post your log**
You can get the log from the intercafe. This is in the Log section.

```
POST LOG HERE
```

**Additional context**
Add any other context about the problem here.

**Comment by schpeen** [comment]
Any updates on this issue? I have tried just about everything but I cannot get the redirect URI to work

**Comment by mkellsy** [comment]
Still need time to set Wink back up so I can test.

**Comment by khuypen** [comment]
an FYI. I got this to work finally, but it was kind of odd.  I used the certified plugin, used the client ID/ Secret, removed the lines about user name/ password, and saved the json. When homebridge restarted, i went to the local address for wink on port 8888 and authenticated with my username and password and then the plugin worked.  This seems to be the v1 way of authenticating thought which seems a bit odd.  if i tried just the user name and password in the wink config, i would get an access type not granted error.  Using both Client Id/ Secret & Username / Password would just get me the error to authenticate with a web browser in the log.
